clab/config.go

```diff
@@ -260,9 +260,17 @@ func (c *CLab) createNodeCfg(nodeName string, nodeDef *types.NodeDefinition, idx
 		// Extras
 		Extras: c.Config.Topology.GetNodeExtras(nodeName),
 	}
+	var err error
+
+	// Load content of the EnvVarFiles
+	envFileContent, err := utils.LoadEnvVarFiles(c.TopoFile.dir, c.Config.Topology.GetNodeEnvFiles(nodeName))
+	if err != nil {
+		return nil, err
+	}
+	// Merge EnvVarFiles content and the existing env variable
+	nodeCfg.Env = utils.MergeStringMaps(envFileContent, nodeCfg.Env)
```

Comment:
Hi @steiler 
let's add a check for env files to be present in a system similar to that code - https://github.com/srl-labs/containerlab/blob/c9f6ddb6c6a04e55ec94b930e970eb67add9bfb2/clab/config.go#L467

There I propose to take it a step further if a path exists, replace the original value of the path with the one that is returned by `utils.ResolvePath` function. That way after doing the check for the path existance you will have your node with the already resolved path. This will remove the need to do the path resolving part that you do in `env.go#L99`.  
Additionally, you won't need to pass c.Topofile.dir to that LoadEnvVarFiles func

Reply:
As discussed offline, utilizing the "verify-Stage" is too late, since these tests are performed after the Nodes are created.
But nevertheless did I shift to using the utils.ResolvePath now.
So please go ahead and check I you think this is now ready to be merged.